targets/fsharp.props

```diff
@@ -7,6 +7,7 @@
     <BuildRevision>170630</BuildRevision>
     <BuildCommand>$(ProjectDirectory)/src/buildfromsource$(ShellExtension)</BuildCommand>
     <NuGetConfigFile>$(ProjectDirectory)/NuGet.Config</NuGetConfigFile>
+    <RepoApiNotImplemented>true</RepoApiNotImplemented>
```

Comment:
FYI- All these updates to the props file for the repo's is going to clash pretty hard with changes I have locally.